lib/author.rb

```diff
@@ -0,0 +1,19 @@
+class Author
+  attr_accessor :first_name, :last_name
+
+  def initialize(first_name: nill, last_name: nill)
```

Comment:
`Nil` is used to represent a null value. In fact, nil is a built-in object in Ruby that represents "nothing" or "no value"
So please change `nill` for `nil` (you write nil with to ll at the end)

Reply:
You also have to receive items as an argument in the def initialize and also id, if you are not sure how to do this I suggest you see label class implementation of Victor